### PR TITLE
Bet 1 Fast Follows Pt 3: Template Library

### PR DESCRIPTION
**Pull request by alexyoung23j**
## Link to Issue
Closes: #3631 and #3633 

## Description of Changes
- Adds the template library section to the contracts page and template preview modal for viewing.
- Also includes copy and style updates to other bet1 pages. 
- This is a follow up to #3536 , which should be merged first. 

## Test Plan
- Tested locally, testing with design on frack soon. 

## Deployment Plan

1. 

## Other Considerations

-

### CODE REVIEW

File: packages/commonwealth/client/scripts/views/pages/contracts/index.tsx [hunk 1]
```diff
@@ -18,6 +29,36 @@ const ContractsPage = () => {
...
+        })
+      );
+    }
+  };
+
+  useEffect(() => {
+    fetchTemplates();
+  }, [contracts, setTemplates]);
+
+  const [tabOn, setTabOn] = useState<'contracts' | 'templates'>('contracts');
```
Comment (masvelio) [inline packages/commonwealth/client/scripts/views/pages/contracts/index.tsx:60]: move usestate up, so all useStates are close to each other

File: packages/commonwealth/client/scripts/views/pages/contracts/index.tsx [hunk 2]
```diff
@@ -29,6 +70,58 @@ const ContractsPage = () => {
...
+          {
+            label: 'Cancel',
+            buttonType: 'mini-black',
+          },
+        ],
+      });
+    }
   };
 
   return (
```
Comment (masvelio) [inline packages/commonwealth/client/scripts/views/pages/contracts/index.tsx:127]: this return is massive and needs some decluttering as it is hard to see what is going on here. Avoid nested ternary

File: packages/commonwealth/client/scripts/views/modals/view_template_modal.tsx [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+import app from 'state';
+import { CWCommunityAvatar } from '../components/component_kit/cw_community_avatar';
+import { User } from '../components/user/user';
+import { TemplateComponents } from '../pages/view_template/view_template';
+import validateType from 'client/scripts/helpers/validateTypes';
+import { CWDivider } from '../components/component_kit/cw_divider';
+import { CWDropdown } from '../components/component_kit/cw_dropdown';
+import { CWTextInput } from '../components/component_kit/cw_text_input';
+
+const renderDisabledTemplate = (form_fields) => {
```
[the remark this answers is not in the document]
Comment (alexyoung23j) [reply packages/commonwealth/client/scripts/views/modals/view_template_modal.tsx:17]: True

File: packages/commonwealth/client/scripts/views/pages/contracts/index.tsx (hunk 1 above)
Comment (alexyoung23j) [reply packages/commonwealth/client/scripts/views/pages/contracts/index.tsx:60]: good catch

**Review by jnaviask — APPROVED** [review]
Low risk.

**Pull request merged**